tools/trap/ramdumpParser.py

```diff
@@ -505,17 +505,53 @@ def unwind_backtrace_with_framepointer(self, fp, sp, lr, pc, ss):
 		self.stacksize = ss
 		self.find_stackframe_using_framepointer(frame)
 
-def format_output(res):
+def format_output(res, string):
      r = res.split('\n')
-     print('symbol addr       :', r[0])
-     print('function name     :', r[1])
-     print('file              :', r[2])
+     print('\t- symbol addr {0}       : {1}'.format(string, r[0]))
+     print('\t- function name {0}     : {1}'.format(string, r[1]))
+     print('\t- file {0}              : {1}'.format(string, r[2]))
+
+# Function to get the number of application binaries, names, text address and sizes
+def find_number_of_binaries(log_file):
+
+    global g_app_idx
+    global g_stext_app
+    global g_etext_app
+
+    # Parse the contents based on tokens in log file.
+    with open(log_file) as searchfile:
+        for line in searchfile:
+            # Get the number of applications loaded
+            if 'elf_show_all_bin_addr:' in line:
```

Comment:
`elf_show_all_bin_addr` is a function name, so it can be changed after sometime.
But it is used more than once.
So how about declaring it as a string at the top of the file?
Even, from `debugsymbolviewer`.